.github/workflows/edge.yml

```diff
@@ -100,60 +100,89 @@ jobs:
       matrix:
         include:
           - os: ubuntu-20.04
-            image: nginx-ingress
+            file: build/Dockerfile
+            image: debian
```

Comment:
''there's an issue with re to test markers, acc .to this only 'ingresses' tests will run on debian and only 'vsr' tests will run on alpine and so on. Think existing matrix will be needed to multiplied by types of images.

Reply:
As discussed offline, we have to make a trade-off on edge because the matrix of jobs would get too large. I'll add two more jobs so that we are testing everything on Debian Plus image, as agreed.